Sources/JWTKeychain/Controllers/API/UserController.swift

```diff
@@ -32,46 +35,48 @@ open class UserController<U: UserAuthenticating>: UserControllerType {
     }
 
     open func register(request: Request) throws -> ResponseRepresentable {
-        let user = try U.makeUser(request: request, hasher: hasher)
+        let user = try userAuthenticator.makeUser(request: request, hasher: hasher)
         let token = try user.createToken(using: signer)
         return try makeResponse(token: token, user: user)
     }
 
-    open func login(request: Request) throws -> ResponseRepresentable {
-        let user = try U.logIn(request: request)
+    open func logIn(request: Request) throws -> ResponseRepresentable {
+        let user = try userAuthenticator.logIn(request: request)
         let token = try user.createToken(using: signer)
         return try makeResponse(token: token, user: user)
     }
 
-    open func logout(request: Request) throws -> ResponseRepresentable {
-        _ = try U.logOut(request: request)
+    open func logOut(request: Request) throws -> ResponseRepresentable {
+        _ = try userAuthenticator.logOut(request: request)
+        // TODO: is it really necessary to return this response? A status code of 200 should suffice.
```

Comment:
Let's always use the `status` key - I'm ok with the `resetPasswordEmail` describing that the instructions has been sent.